src/tqec/templates/orchestrator.py

```diff
@@ -110,7 +110,7 @@ def add_template(
         self._templates.append(template_to_insert.template)
         self._relative_position_graph.add_node(template_id, plaquette_indices=indices)
         self._maximum_plaquette_mapping_index = max(
-            self._maximum_plaquette_mapping_index, max(indices)
+            [self._maximum_plaquette_mapping_index] + indices
```

Comment:
This is a style preference, but I am not sure creating a whole new list is the best way to compute the maximum. Also, I find the previous version easier to read, but again, personal preference.

Reply:
I just changed it, as pylint suggested not using nested max calls

Reply:
Who am I to contradict pylint ^^ I do not have that warning, but I am 100% following pylint, so I agree with your version. Do you have a link to the rule? I am interested by the rationale behind this pylint check.

Reply:
It's https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/nested-min-max.html#nested-min-max-w3301. I don't know if I get it because it can't infer the type?